src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -58,7 +59,7 @@ public void createWorkspaceWithDefaults(String authToken, UUID id) {
     }
   }
 
-  public void deleteWorkspace(String authToken, UUID id) {
+  public void deleteWorkspace(String authToken, UUID id) throws SamResourceNotFoundException {
```

Comment:
I don't think this is throwing the exception anymore. let's remove the checked exception from the method, and I think the exception file itself.

Reply:
Yep, good catch.